aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -1287,6 +1287,25 @@ struct HelperInterpCubic : public HelperInterpBase {
 #undef a
   }
 
+  template<typename scalar_t>
+  static inline scalar_t aa_filter_075(scalar_t x) {
+    // https://en.wikipedia.org/wiki/Bicubic_interpolation#Bicubic_convolution_algorithm
+    // In this code we are using alternative a=-0.75 definition that matches
+    // get_cubic_upsample_coefficients implementation, as used by float path
+    // TODO: merge both implementations, the only difference is the value of a.
+    constexpr scalar_t a = -0.75;
+    if (x < 0.0) {
+      x = -x;
+    }
```

Comment:
Why not use `std::abs` here?
```suggestion
    x = std::abs(x);
```